meow/geometries.py

```diff
@@ -153,7 +153,7 @@ def _mask2d_axis_x(self, X, Y, z):
                 return np.zeros_like(Y, dtype=bool)
             intersections = sg.MultiLineString([intersections])
 
-        mask = np.zeros_like(Y, dtype=bool)
+        np.zeros_like(Y, dtype=bool)
```

Comment:
I think we can fully leave this out

Reply:
we can't. I'm not even sure why this line was changed. As you can see mask needs to be defined for the line

```python
            mask |= (x_min <= X) & (X <= x_max) & (y_min <= Y) & (Y <= y_max)
```

to work.